src/tags-input.less

```diff
@@ -1,5 +1,6 @@
 .tags-input {
-	display: inline-block;
+	display: inline-flex;
+	flex-wrap: wrap;
```

Comment:
seems like this would need a `min-width` if it were to match the current implementation, right?